admin_guide/tcp_ingress_external_ports.adoc

```diff
@@ -0,0 +1,99 @@
+[[admin-guide-unique-external-ips-ingress-traffic]]
+= Assigning Unique External IPs for Ingress Traffic
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+Cluster administrators can assign a unique IP address to a service that, if
+routed correctly, enables external traffic to reach the ports of that service.
+This external IP assignment allows you to expose an external port, enabling
+ingress traffic (TCP/UDP) routing to a definable backend. This is much simpler
+than trying to manage the port space of a limited number of shared IP addresses.
+
+There is support for both automatic and manual assignment of IP addresses, each
+address is guaranteed to be assigned to a limit of not more than one service.
+This ensures that each service can simply direct traffic from its chosen ports,
+regardless of the ports selected by other services.
+
+[[unique-external-ips-ingress-traffic-restrictions]]
+== Restrictions
+
+To use `*IngressIP*`, you can:
```

Comment:
The 2 statements that follow better describe 'External IP'. An 'ingress ip' is an external ip guaranteed to be uniquely assigned to a service.  A manually-assigned external ip has no such guarantee.

'To use an ExternalIP...' ?